src/wstool/multiproject_cmd.py

```diff
@@ -274,6 +276,32 @@ def do_work(self):
     return outputs
 
 
+def cmd_foreach(config, command, shell=False):
+    """Run command in all SCM entries in config, relative to path"""
+
+    class ForeachRetriever(object):
+        def __init__(self, element, command, shell):
+            self.element = element
+            self.command = command
+            self.shell = shell
+
+        def do_work(self):
+            command = self.command
+            if not self.shell:
+                command = shlex.split(command)
+            po = Popen(command, cwd=self.element.path, shell=self.shell,
```

Comment:
run_shell_command() has too many options and not sure how to handle no-stdout, no-stderr.
The command print out the stdout and stderr in that command. (using `print()` function not returns the outputs)